### DISCUSSION TITLE
Tailwind styles not working with lazy-blocks plugin.

### DISCUSSION THREAD

**Discussion by tavo379** [gregsullivan/_tw#129] [Q&A]
Hey guys, I'm here to see if anyone has any idea how to solve the problem I have, basically for ease I use the lazy-blocks plugin, I have 2 problems with the plugin
1- This is not so important but if anyone knows how to answer me I'd appreciate it, how can I make the editor use js, for example when I create a custom block with swiper, in the editor it looks like a list, the js and swiper styles are not applied so it is displayed incorrectly.
2- This is the main error, basically when using lazy-blocks not all the tailwind styles that I add are applied, it is peculiar because some classes are displayed correctly and others are not, which I do not understand.
Below I leave an example, the ml-9 class is not applied, but the flex and bg-black container classes below are applied correctly.

![image](https://github.com/user-attachments/assets/89b36593-afbe-4daa-9a44-0387db6b910f)

**Note:**
the path of the blocks is /theme/block/block-name/block.php

**Comment by gregsullivan** [comment]
Hi Gustavo—Thanks for posting this!

It seems like you posted a second thread about the JavaScript side of things, so I'll address that part of your question in that thread.

In terms of the styles: Are you adding all of them in the `block.php` file, or are some being added within the WordPress admin? I know that Lazy Blocks advertises being able to "rapidly build custom blocks without ever leaving your WordPress editor." That might not align with the requirements of _tw in terms of generating the CSS locally. If you can ensure all of the CSS classes are in a PHP file inside of your theme, it should work.

If you can provide more details about where you're adding these classes, I'll have a better sense of where the issue might lie!
